tests/roles/cinder_adoption/tasks/cinder_volume_netapp_nfs.yaml

```diff
@@ -0,0 +1,50 @@
+- name: Deploy Podified Cinder - Netapp
+  block:
+    - name: Slurp cinder.conf from controller
+      become: true
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ controller_ssh }}
+        CONTROLLER1_SCP="{{ controller1_ssh  | regex_replace('^ssh', 'scp')}}"
+        ${CONTROLLER1_SCP}:{{ cinder_tripleo_path }} {{ cinder_conf_path }}
+        ${CONTROLLER1_SCP}:{{ tripleo_nfs_shares_config_path }} {{ nfs_shares_conf_path }}
+
+    - name: Extract Netapp data from cinder.conf
+      ansible.builtin.set_fact:
+        netapp_config: "{{ netapp_config | default({}) | combine({item: lookup('ansible.builtin.ini', item, file=cinder_conf_path, section=cinder_netapp_backend, allow_no_value=True)}) }}"
```

Comment:
These variables are being set as facts. We should not set them as fact if possible, because the ansible namespace is global.
A first step is to use a less generic name. cinder_netapp_config.